blockchain/contracts/Staking.txt

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MIT
```

Comment:
What is this file? I notice it is a text file not a solidity contract

Reply:
Kind of pseudo-contract I started with, didn't want to lose the logic and it ended up being commited, can be disregarded